finalPrep/modZeroHero.js

```diff
@@ -55,30 +96,72 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
+var scaryMonster = {
+  name: "Godzilla",
+  smell: "Stinky",
+  weight: 500,
+  citiesDestroyed: ["Cancun", "Miami"],
+  luckyNumbers: [7, 21, 3],
+  address: {
+    number: 12,
+    street: "16th Street",
+    state: "Kansas",
+    zip: 80203
+  }
+}
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
+class SuperHero {
+  constructor(){
```

Comment:
The instructions here ask you to create a SuperHero class with three DYNAMIC values - `name`, `superpower` and `age`.  These dynamic values would need to be passed in as arguments and assigned in the `constructor` method.